versioned-lambda/orb.yml

```diff
@@ -59,11 +59,24 @@ jobs:
             - run:
                 name: Sudo install
                 command: yum install -y sudo
-            - snyk/scan:
-                token-variable: SNYK_TOKEN
-                monitor-on-build: true
-                severity-threshold: low
-                project: ${CIRCLE_PROJECT_REPONAME}/${CIRCLE_BRANCH}
+            - when:
+                condition:
+                  equal: [master, $CIRCLE_BRANCH]
```

Comment:
I think you need to reference `<< pipeline.git.branch >>` or whatever it was in the apollo extract repo. I don't think env vars are interpolated when it evaluates the when blocks